common/utils/src/main/resources/error/error-classes.json

```
@@ -2261,6 +2273,12 @@
     },
     "sqlState" : "42000"
   },
+  "INVALID_STATEMENT_FOR_EXECUTE_INTO" : {
+    "message" : [
+      "The INTO clause of EXECUTE IMMEDIATE is only valid for queries but the given statement isn't: <sqlString>."
+    ],
+    "sqlState" : "42606"
```

"An invalid hexadecimal constant has been detected"?
How about:
"The option specified on PREPARE or EXECUTE is not valid."
(07 is all about dynamic SQL)
```suggestion
    "sqlState" : "07501"
```